server/resolvers/browsersResolver.js

```diff
@@ -1,9 +1,5 @@
-const { getBrowsers } = require('../models/services/BrowserService');
-
-const browsersResolver = () => {
-    return getBrowsers(undefined, undefined, undefined, undefined, undefined, {
-        order: [['name', 'asc']]
-    });
+const browserResolver = async (_, __, context) => {
```

Comment:
Minor nit: It seems the standard has been to keep these resolver functions as having the same name as their file names which is sometimes useful for doing quick searches.

Could this function be renamed back to `browsersResolver`?

Reply:
Ok. Done!